ext/session/mod_user.c

```diff
@@ -75,7 +75,15 @@ PS_OPEN_FUNC(user)
 	zval args[2];
 	STDVARS;
 
+	if (PS(in_save_handler)) {
+		PS(in_save_handler) = 0;
+		php_error_docref(NULL, E_WARNING, "Cannot call save handler function recursive manner");
```

Comment:
This should be:

"Cannot call session save handler recursively or "Cannot call session save handler in a recursive manner"

I took the liberty to add 'session', as it eases up where to look during debugging quickly